### PR TITLE
Add seccomp build tag

### PR DESCRIPTION
**Pull request by crosbymichael**
Add a seccomp build tag and also support in the Makefile to add or
remove build tags.

Signed-off-by: Michael Crosby <crosbymichael@gmail.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -20,7 +25,12 @@ test: runctestimage
 	docker run -e TESTFLAGS --privileged --rm -v $(CURDIR):/go/src/$(PROJECT) $(RUNC_TEST_IMAGE) make localtest
 
 localtest:
+ifdef BUILD_TAGS
+	go test -tags $(BUILD_TAGS) ${TESTFLAGS} -v ./...
```
Comment (LK4D4) [inline Makefile]: I think it's simpler to use just `"$(BUILD_TAGS)"`, `-tags ""` is valid syntax.

File: Makefile [hunk 2]
```diff
@@ -1,10 +1,15 @@
 RUNC_TEST_IMAGE=runc_test
 PROJECT=github.com/opencontainers/runc
 TEST_DOCKERFILE=script/test_Dockerfile
+BUILD_TAGS=seccomp
```
Comment (LK4D4) [inline Makefile]: maybe `BUILDTAGS` to be consistent with `TESTFLAGS`, also underscore is superhard to type :)

**Comment by mheon** [comment]
Just to make sure you're aware, this will break Godep - it will no longer detect the libseccomp-golang repository as necessary as every use is behind the seccomp tag, and will refuse to vendor it.

**Comment by crosbymichael** [comment]
why godeps why!

**Comment by crosbymichael** [comment]
@LK4D4 updated

**Comment by hqhq** [comment]
what about some docs in README.md? Default make will still fail on most distros.

**Comment by aidanhs** [comment]
+1 to getting this in given it lets me use runc on Ubuntu 12.04.

**Comment by LK4D4** [comment]
LGTM

**Pull request merged**